source/gui/_addonStoreGui/viewModels/store.py

```diff
@@ -241,6 +241,21 @@ def _makeActionsList(self):
 				validCheck=lambda aVM: isinstance(aVM.model, _AddonStoreModel),
 				actionTarget=selectedListItem
 			),
+			AddonActionVM(
+				# Translators: Label for an action that opens the webpage to see and send feedback for the selected add-on
+				displayName=pgettext("addonStore", "See or provide &feedback"),
+				actionHandler=lambda aVM: startfile(
+					cast(
+						str,
+						cast(_AddonStoreModel, aVM.model).reviewUrl

```

Comment:
See my previous suggestion about changing the capitalization of these to "reviewURL", for consistency with other "*URL" variables and fields.